src/components/SuggestionsList/SuggestionsList.tsx

```diff
@@ -68,16 +70,16 @@ export default class SuggestionsList extends React.PureComponent<SuggestionsList
       <div className={join(['y-suggestions-list', 'y-hc-border', className, classNames.root])}>
         {hasResults && (
```

Comment:
never a fan of inline logic in JSX - cant tell from this snippet - should the component not just render `null` if !hasResults, or if not something like `const results = hasResults ? this.getResults() : null;`

Reply:
ya i'll look at these

Reply:
done